index.html

```diff
@@ -632,7 +629,7 @@
 (eg. triggering a form control that is not on a modal dialog) and if that [=consent=] is persistent,
 there should be a vivid indicator that data is being transmitted shown at all times, in such a way
 that the person can easily switch it off. In general, providing [=consent=] should be rare,
-difficult, highly intentional, and temporary.
+slow, highly intentional, and temporary.
```

Comment:
We could just drop "slow", and let any necessary slowness be implied by "highly intentional"?